### ISSUE TITLE
Journal-wide UI for tracking deposits and distribution

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#5980]
**Describe the problem you would like to solve**
Journals deposit and distribute published material to many different third-party vendors, such as Crossref, DataCite, DOAJ and ORCID. In addition, journals provide data about their publications through an OAI endpoint and may in the future need to publish cross-journal metadata such as licensing details.

Where OJS supports automated deposits, it does so on an ad-hoc basis. This is usually managed through the Tools > Import/Export interface. There is no single point where a journal manager can get an overview of their distribution tools, identify outstanding distribution tasks, and rectify problems with distributed data.

**Describe the solution you'd like**
A single page where different tools and services could load a dashboard panel, provide information about outstanding tasks, and link the user on to settings and tools that may be needed.

Deposit and distribution tools and services that might be added to this page include:

- Crossref
- DataCite
- Pubmed
- DOAJ
- ORCID
- Preservation Network (PN)
- CLOCKSS / LOCKSS
- Institutional Repositories
- OAI
- PKP's Beacon

Plugins could add their own dashboard panel when active and the page could provide guidance to journal managers on what services may be appropriate for their needs.

**Who is asking for this feature?**
We get persistent requests for improving the Crossref deposit management, but these same issues arise for other services. In addition, giving these services greater visibility within the application may help communicate the value of using OJS as a publishing tool. 

**Additional information**
This is a preliminary mockup demonstrating how tools and services could be added and could provide their own status information, quick access to frequent tasks, etc.

![distribution](https://user-images.githubusercontent.com/2306629/84262897-5d3b0280-ab16-11ea-8d49-87aec5daef9b.png)

**Comment by AhemNason** [comment]
Just echoing James feedback here again. I think this is great. And I think "distribution" is definitely a better way to think about these things than import/export.

**Comment by willinsky** [comment]
I agree with James and Mike on this being such a great addition for managing journals. I'd also like to consider whether it could be a part of our "journal integrity" initiative, which is intended to help journals do more to establish for authors, researcher-readers, and the public the steps they've taken to publish to the highest academic standards in preserving the journal's integrity as a source of knowledge. 

In the case of this new Distribution feature, for each service and tool that a journal was operating (if not always complete), OJS could provide a tastefully designed "Journal Services" box on the journal's homepage (perhaps as an option for each service or as a whole), which would include a list with a text along these lines, "[title of journal] distributes article and journal data to the following scholarly publishing services: DOAJ, CrossRef, ORCID, PKP Preservation Network, PKP Beacon, Google Scholar..." (with hyperlinks to each). 

Now I realize that there is not going to ever be a fool-proof integrity indicator. Yet part of this effort is for us to take more of a lead with OJS in demonstrating to the public what journals do in their efforts to maintain high academic standards. A big step in that direction might be achieved, I want to believe, by three steps: Displaying these services on the journal's homepage, in addition to providing on article landing pages the peer-review dates and reviewer numbers (with content optionally dependent on authors and reviewers), as well as displaying ORCIDs for all editors and editorial board members (by having an optional "board" generator to which journal managers can add users with ORICIDs for display under About).

**Comment by NateWr** [comment]
A more detailed use-case from @davidjb for an ORCID management UI in this [comment](https://github.com/pkp/pkp-lib/issues/6971#issuecomment-827298888):

> In my situation, I've currently got a multi-journal installation where some journals are looking at official ORCID integration and others aren't. Because the OJS install uses federated single-sign on (Shibboleth), this means that one journal someone contributes to may authorise ORCID iDs and another may allow manual entry (e.g. have the ability to edit their ORCID iD in a text box at the site-level or on a different journal) -- having this all be consistent (e.g. what's described above) would solve this issue.
> 
> The ability to manage and see authorised/unauthorised ORCID iDs (like what @NateWr, @withanage and #5980 describe) would be fantastic and a huge win for journal managers. At present, the visibility isn't there (beyond looking at published articles) and trying to retrospectively link ORCID iDs means an un-publish/edit contributor/re-publish cycle for all articles and all contributors on each article -- a long and complicated process.

**Comment by mreiko** [comment]
@Devika008 Confirming that design is now complete and in the hands of the dev team as we build out the distribution/deposit functionality?

**Comment by Devika008** [comment]
hello @mreiko,

Eric and Bozana have to give me a go ahead if everything in the design is proper or if I need to make any modification to it

**Comment by mreiko** [comment]
@asmecher @Devika008 I understand we're not doing this one anymore. Can you please add notes on your last discussion and rationale?

**Comment by asmecher** [comment]
@Devika008, correct me if I'm wrong... My recollection is that we kicked around a number of designs and found that the number of common elements that we could present for each of the indexing services was extremely limited, leading to a situation where each one needed a large, personalized tabular interface with status, pagination, etc. Some listed content by publication, some by submission; status details and available actions varied widely. Having all of this on one megapage was unwieldy. On the flip side, reducing it below the level of listing individual content elements left little beyond a list of links to where the indexing services were managed elsewhere (e.g. Tools > Importexport).

**Comment by AhemNason** [comment]
I'll note here for posterity that this has _some kind of alignment_ with some of the Craft OA plugins based around indexing. From Radek & Co at Munin. https://septentrio.uit.no/index.php/SCS/article/view/8304 

Specifically this Disco plugin, which is a different vibe, I acknowledge. https://www.craft-oa.eu/ojs-diamond-plugins/ 

I think something of an early warning system here is nice and it's really good for editors to know. In _particular_, one of the biggest issues run into via hosting is that when an editorial changes, the new editors don't always know what's connected to what. This can be managed a bit when _people communicate_. But I do think _some method_ of tracking editorial or publishing history is useful as we get older and older titles on OJS.
